From b7be34e4c7183817729e9e23eba32d93fceaf34d Mon Sep 17 00:00:00 2001
From: "lauren n. liberda" <lauren@selfisekai.rocks>
Date: Sat, 20 Apr 2024 22:25:14 +0200
Subject: [PATCH] zig 0.12: std.ChildProcess.exec -> run

solves "error: struct 'child_process.ChildProcess' has no member named 'exec'"
---
 src/gitversiontag.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gitversiontag.zig b/src/gitversiontag.zig
index 40aed0f..08480a7 100644
--- a/src/gitversiontag.zig
+++ b/src/gitversiontag.zig
@@ -6,7 +6,7 @@ pub fn gitVersionTag(a: std.mem.Allocator) []const u8 {
         "--sort=-creatordate",
     };
 
-    if (std.ChildProcess.exec(.{ .argv = args[0..], .allocator = a })) |ret| {
+    if (std.ChildProcess.run(.{ .argv = args[0..], .allocator = a })) |ret| {
         var it = std.mem.split(u8, ret.stdout, "\n");
         if (it.next()) |firstline| {
             return firstline;
